controller/handler/trial.js

```diff
@@ -124,6 +127,10 @@ function trialView (request, reply) {
 
             const endDate = processRules(ruleValues, Date.now());
 
+            if (request.query.requestType === 'ajaxRequest') {
```

Comment:
This line of code is also causing the tests in `controller/handler-test/trial.js` to fail.